admin/class-metabox.php

```diff
@@ -1210,6 +1218,31 @@ function column_sort_orderby( $vars ) {
 	}
 
 	/**
+	 * Hide the SEO Title, Meta Desc and Focus KW columns if the user hasn't chosen which columns to hide
+	 *
+	 * @param array|false $result
+	 * @param string      $option
+	 * @param WP_User     $user
+	 *
+	 * @return array|false $result
+	 */
+	public function column_hidden( $result, $option, $user ) {
+		global $wpdb;
+
+		$prefix = $wpdb->get_blog_prefix();
+		if ( ! $user->has_prop( $prefix . $option ) && ! $user->has_prop( $option ) ) {
+
+			if ( ! is_array( $result ) ) {
```

Comment:
Please put this `is_array` check outside the if. This in case when there isn't a valid array passed and the user won't come inside this if block.